CHANGELOG.md

```diff
@@ -13,6 +13,20 @@ Release dates of 3.x versions:
 - 3.9: 2018-01-13
 - 3.10: 2018-11-24
 
+3.11.0 beta 1 (2020-02-18)
```

Comment:
should be formatted like the headers below, and i'd personally not use "beta 1" since you'll have to keep updating it for every pre-release bump. the changelog is more there to describe the actual releases than pre-release increments, IMO. similar to how the schelp doc is called "news in 3.11" and not "news in 3.11-beta1"